skimage/segmentation/_join.py

```diff
@@ -114,20 +116,29 @@ def relabel_sequential(label_field, offset=1):
     >>> relab
     array([5, 5, 6, 6, 7, 9, 8])
     """
+    offset = int(offset)
+    if offset <= 0:
+        raise ValueError("Offset must be strictly positive.")
```

Comment:
Nice one, yeah the error message was non-sensical before
```python
In [14]: relabel_sequential(np.asarray([2, 3, 4]), -1)                                                  
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-14-e5025e3e4264> in <module>
----> 1 relabel_sequential(np.asarray([2, 3, 4]), -1)

~/miniconda3/envs/owl/lib/python3.7/site-packages/skimage/segmentation/_join.py in relabel_sequential(label_field, offset)
    129         labels = np.concatenate(([0], labels))
    130     inverse_map = np.zeros(offset - 1 + len(labels), dtype=np.intp)
--> 131     inverse_map[(offset - 1):] = labels
    132     relabeled = forward_map[label_field]
    133     return relabeled, forward_map, inverse_map

ValueError: could not broadcast input array from shape (4) into shape (2)
```

Reply:
:+1: